articles/active-directory-b2c/service-limits.md

```diff
@@ -169,8 +169,10 @@ The following table lists the administrative configuration limits in the Azure A
 |Number of sign-out URLs per application        |1          |
 |String Limit per Attribute      |250 Chars          |
 |Number of B2C tenants per subscription      |20         |
-|Total number of objects (user accounts and applications) per tenant (default limit)|1.25 million |
-|Total number of objects (user accounts and applications) per tenant (using a verified custom domain). If you want to increase this limit, please contact [Microsoft Support](find-help-open-support-ticket.md).|5.25 million |
+|Total number of objects per tenant (default limit)|1.25 million (1M user accounts and 250K applications) |
```

Comment:
Can we maintain the _1.25 million_ in the Limit column, then add notes below the table (reference using footnotes). How this work out? We want to keep the Limit column clean. We can then apply all this in the other new row.